src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -157,7 +218,7 @@ export class HttpConnection implements IConnection {
                     // No fallback or negotiate in this case.
                     await this.transport!.connect(url, transferFormat);
                 } else {
-                    return Promise.reject(new Error("Negotiation can only be skipped when using the WebSocket transport directly."));
```

Comment:
Why did you change these? Shouldn't these continue to just reject the promise instead of throwing?

Reply:
https://github.com/aspnet/AspNetCore/pull/8566#discussion_r272728004